packages/monitor-v2/src/llm-bot/examples.ts

```diff
@@ -108,6 +114,18 @@ class Strategy extends LLMStrategy<OptimisticOracleRequestPolymarket, Optimistic
   }
 }
 
+class StrategyHandler extends OptimisticOracleStrategyHandler<OptimisticOracleRequestPolymarketResult> {
+  process = async () => {
```

Comment:
It is bit confusing that this extended class has process without parameters while the abstract base class had StrategyHandlerOptions parameter)